live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var str = 'Mozilla';
+
+console.log(str.substr(1, 2));
+// expected output: 'oz'
```

Comment:
I know, super-nitpicky, but please use double quotes in expected output, to match the actual output.

Reply:
Fixed the double quotes.